reason-react-native/src/apis/YellowBox.md

```diff
@@ -1,13 +1,23 @@
 ---
 id: apis/YellowBox
 title: YellowBox
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/debugging#warnings
 ---
 
-```reason
-[@bs.module "react-native"] [@bs.scope "YellowBox"]
-external ignoreWarnings: array(string) => unit = "";
+Warning messages are displayed in white on a yellow background; which is known
+as a YellowBox. Warnings may be trigged with the `warn` method already included
```

Comment:
I would go for ``Warnings may be trigged with the `Js.Console.warn` method included BuckleScript`` (so you can directly copy/paste/visualise the call to make

Reply:
I would write ```Warnings may be trigged with the `Js.Console.warn` function included in BuckleScript.```